src/main/scala/cognite/spark/v1/DataModelInstancesRelation.scala

```diff
@@ -61,16 +61,16 @@ class DataModelInstanceRelation(
     if (modelType == DataModelType.EdgeType) {
       modelProps ++ Map(
         "externalId" -> DataModelPropertyDefinition(PropertyType.Text, false),
-        "type" -> DataModelPropertyDefinition(PropertyType.Text, false),
+        "nodeType" -> DataModelPropertyDefinition(PropertyType.Text, false),
```

Comment:
edgeType?

Reply:
true and now I am not sure if this is the fix :D